TrafficCapture/dockerSolution/src/main/docker/k8sConfigMapUtilScripts/README.md

```diff
@@ -0,0 +1,33 @@
+# Kubernetes Config Map Utility Library
+
```

Comment:
Whats the lifecycle of this tool, it looks like it should be turned on and it watches for changes, or should it be triggered on /off by something?